R/fparse.R

```diff
@@ -181,17 +181,17 @@
 #'        single_null = NA_real_)
 #'
 #' # handling invalid JSON and parsing errors ==================================
-#' fparse("junk JSON", parse_error_ok = TRUE)
-#' fparse("junk JSON", parse_error_ok = TRUE,
-#'        on_parse_error = "can't parse invalid JSON")
-#' fparse(
-#'     c(junk_JSON_1 = "junk JSON 1",
-#'       valid_JSON_1 = '"this is valid JSON"',
-#'       junk_JSON_2 = "junk JSON 2",
-#'       valid_JSON_2 = '"this is also valid JSON"'),
-#'     parse_error_ok = TRUE,
-#'     on_parse_error = NA
-#' )
+#' # fparse("junk JSON", parse_error_ok = TRUE)
+#' # fparse("junk JSON", parse_error_ok = TRUE,
+#' #       on_parse_error = "can't parse invalid JSON")
+#' # fparse(
+#' #    c(junk_JSON_1 = "junk JSON 1",
+#' #      valid_JSON_1 = '"this is valid JSON"',
+#' #      junk_JSON_2 = "junk JSON 2",
+#' #      valid_JSON_2 = '"this is also valid JSON"'),
+#' #    parse_error_ok = TRUE,
+#' #    on_parse_error = NA
+#' #)
```

Comment:
That looks like it would break roxygen.   Did you mean to comment this out?   (R also has \dontrun{ ....} for this.)

Reply:
I commented this out to disable the test because it was an issue of On Demand not recognizing the broken JSON as I have discussed above, but I was not aware that I could do \dontrun for this, so you should probably change it.